### PR TITLE
Removes Thermobaric from Requisitions

### PR DESCRIPTION
**Pull request by Chaznoodles**
## About The Pull Request

Removes Thermobaric and it's ammo from Requisitions

## Why It's Good For The Game

When I first redid Requisitions pricing and added the thermobaric at 250 points, it was added as a joke because if Req was able to stack up 250 points, it meant xenos were on the run/had no map control and the round was being dragged out. With the implementation of the Tadpole and miner modules, this became a far more achievable goal, so I'm removing the one-shot kill meme cannon.

## Changelog
:cl:
tweak: Due to demand, the Underdecks has ran dry of Thermobaric Launchers.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -14. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Jeff-jefferson-jeffery** [comment]
just make it 750 like tank
"""improve""" don't remove

**Comment by BeloneX** [comment]
> just make it 750 like tank
> """improve""" don't remove

Remove don't improve.

**Comment by Chaznoodles** [comment]
> just make it 750 like tank
> """improve""" don't remove

I sentence you to balls on face

**Comment by SilencedMP5A5** [comment]
bruh leave sandbags alone, lol

**Comment by Chaznoodles** [comment]
> bruh leave sandbags alone, lol

image

Sandbags are the best choice right now because people kept nerfing metal cades without considering sandbags. This brings their Requistions cost/number more in line with their usefulness. If people start to misuse the fact that they come in a stack in construction pouches vended instead, I'll just drop their HP.

**Comment by Losenis** [comment]
You mean the sandbags that still need additional sandbags to be used for repair, rather than just a blowtorch and welderpack for no additional req points? The sandbags that are pretty muched fucked in ice maps and prison station unless you get to a very specific spot xenos could camp at? The ones that are a pita to carry filled compared to carrying ~12 metal cades/~8 wired metal cades in a stack, even worsening the earlier point? The ones that you still need to get metal for wiring, and plasteel to make a gate unless you want to make that cadeline impassable for marines too unless you invest in a maze?

Maybe just make the overglorified vertical car engine covers not take half a round to set up for their paper HP and you'll see them used more often, given you don't have to care as much about the logistic math of getting a good ratio of metal to sandbags if you just spam metal and use it all for cades, upgrades, wiring, and razorwiring.

**Comment by MLGTASTICa** [comment]
Instead of halfing sandbags per order perhaps consider  maybe modifying their integrity to reflect properly?

**Comment by Chaznoodles** [comment]
> Instead of halfing sandbags per order perhaps consider maybe modifying their integrity to reflect properly?

When I redid req pricing a while back, I was under the impression a full stack of sandbags was 25 as that's what it comes in _every_ other sandbag pile. It's 50. Now you get 25.

> You mean the sandbags that still need additional sandbags to be used for repair, rather than just a blowtorch and welderpack for no additional req points? The sandbags that are pretty muched fucked in ice maps and prison station unless you get to a very specific spot xenos could camp at? The ones that are a pita to carry filled compared to carrying ~12 metal cades/~8 wired metal cades in a stack, even worsening the earlier point? The ones that you still need to get metal for wiring, and plasteel to make a gate unless you want to make that cadeline impassable for marines too unless you invest in a maze?
> 
> Maybe just make the overglorified vertical car engine covers not take half a round to set up for their paper HP and you'll see them used more often, given you don't have to care as much about the logistic math of getting a good ratio of metal to sandbags if you just spam metal and use it all for cades, upgrades, wiring, and razorwiring.

Such is the tradeoff of sandbags. No skill needed, fast to set up. I ain't here to buff metal cades chief, I'm here to make sandbags cost more.

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -1050,7 +1040,7 @@ ENGINEERING
 
 /datum/supply_packs/engineering/sandbags
 	name = "50 empty sandbags"
-	contains = list(/obj/item/stack/sandbags_empty/full)
+	contains = list(/obj/item/stack/sandbags_empty/half)
 	cost = 10
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:1044]: A) You didnt update description
B) Atomize this

Comment (Chaznoodles) [reply code/modules/reqs/supplypacks.dm:1044]: Haha atomisation go brrr, will update tomorrow

**Comment by Chaznoodles** [comment]
#5656 atomised

**Review by TiviPlus — APPROVED** [review]

**Comment by Walarks** [comment]
I'd be ok with this if thermobaric gets replaced by tank.

**Comment by Chaznoodles** [comment]
> I'd be ok with this if thermobaric gets replaced by tank.

No

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-20). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**